components/frontend/src/dashboard/CardDashboard.js

```
@@ -36,9 +36,9 @@ export function CardDashboard({ cards, initialLayout, saveLayout }) {
     const [dragging, setDragging] = useState(false)
     useEffect(() => {
         const dashboard = document.getElementById("dashboard")
-        Promise.all(dashboard.getAnimations().map((animation) => animation.finished)).then(
-            () => dashboard.classList.add("animated"), // Used by the renderer to wait for animations to finish
-        )
+        Promise.all(dashboard.getAnimations().map((animation) => animation.finished))
+            .then(() => dashboard.classList.add("animated")) // The animated class is used by the renderer to wait for animations to finish
+            .catch((error) => console.log(error))
```

Why not `showMessage()` here, is silent fail ok?

---

Because this is mainly for the renderer and the toaster would get included in the PDF.